featuretools/tests/entityset_tests/test_es.py

```diff
@@ -2461,3 +2461,43 @@ def test_setitem(add_dataframe):
     es["new_df"] = df
     assert add_dataframe.called
     add_dataframe.assert_called_with(dataframe=df, dataframe_name="new_df")
+
+
+def test_latlong_nan_normalization(latlong_df):
+    latlong_df.ww.init(
+        name="latLong", index="idx", logical_types={"latLong": "LatLong"}
+    )
+    latlong_df.sort_values(by="idx", inplace=True)
+
+    dataframes = {"latLong": (latlong_df,)}
+
+    relationships = []
+
+    es = EntitySet("latlong-test", dataframes, relationships)
+
+    normalized_df = to_pandas(es["latLong"])
+
+    expected_df = pd.DataFrame(
+        {"idx": [0, 1, 2], "latLong": [(np.nan, np.nan), (1, 2), (np.nan, np.nan)]}
+    )
+
+    pd.testing.assert_frame_equal(normalized_df, expected_df)
```

Comment:
It's interesting to me that for the spark test were are comparing these two dataframes, but `pd.testing_assert_frame_equal` considers them equal. I guess that's OK?

```
(Pdb) normalized_df
   idx       latLong
0    0  [None, None]
1    1    [1.0, 2.0]
2    2  [None, None]
(Pdb) expected_df
   idx     latLong
0    0  (nan, nan)
1    1      (1, 2)
2    2  (nan, nan)
(Pdb) pd.testing.assert_frame_equal(normalized_df, expected_df)
(Pdb) 
```

Reply:
I think this comes down to the inconsistent way python compares NaNs especially when they are in tuples and lists. I can add more checks, but I think it might be overkill

Reply:
I'm not overly concerned about it because for our purposes those two are equal. More concerned that we would get something that the assertion things is equal when they are really unequal. As long as we don't get false positives here, I think we are ok.